Makefile

```diff
@@ -28,6 +28,10 @@ test:	verify
 release-local:
 	MANIFESTS=$(shell mktemp -d) hack/release-local.sh
 
+.PHONY: test-e2e
+test-e2e:
+	KUBERNETES_CONFIG="$(KUBECONFIG)" go test -v -tags e2e ./...
```

Comment:
Am ok with this but it seems like for consistency, maybe put this into `hack/test-end-to-end.sh` and invoke it?